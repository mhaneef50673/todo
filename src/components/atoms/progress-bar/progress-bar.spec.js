import React from 'react';
import { mount } from 'enzyme';
import { mountToJson} from 'enzyme-to-json';
import ProgressBar from './index';

describe('ProgressBar Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <ProgressBar percentage={10} />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});