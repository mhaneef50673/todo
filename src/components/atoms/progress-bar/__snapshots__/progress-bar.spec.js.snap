// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Component should render the component 1`] = `
<ProgressBar
  percentage={10}
>
  <div
    className="progress-bar"
  >
    <div
      className="filler"
      style={
        Object {
          "width": "10%",
        }
      }
    />
    <div
      className="loadingText"
    >
      <span>
        Loading...
      </span>
    </div>
  </div>
</ProgressBar>
`;
